build_scripts/deploy.sh

```diff
@@ -1 +1,2 @@
+echo "Calling webhook to trigger app restart: $1"
```

Comment:
The webhook is secret, it contains the deployment key for the App Service! Travis logs are public e.g. https://travis-ci.com/github/cc1000/DevOps-Course-Starter/builds/227703742#L2273

Remember to rotate the password once this is fixed (on the Azure portal go to the App Service -> Container Settings -> FTPS credentials -> Application scope).

Reply:
Done.